students/biegon_piotr/lesson_02_flow_control/lost_card.py

```diff
@@ -0,0 +1,17 @@
+print("Lost card\n")
+
+N = int(input("Enter the number of cards: "))
+
+x = 0
+y = 0
+
+for i in range(1, N):
+    n = int(input("Enter the value of the card: "))
+    x += n
+
+for i in range(1, N + 1):
```

Comment:
Instead of iterating again over a range of values could combine those two `for` loops into single one 
or
use built-in sum function e.g.`y = sum(range(N+1))`